service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -119,13 +102,14 @@ impl GenerateProtocol for RestJsonGenerator {
         writeln!(writer, "use serde_json;")?;
 
         // avoid unused imports when building services that don't use params
-        if service.service_type_name() != "Batch" {
+        if service_has_query_parameters(service) {
```

Comment:
I like the idea of just doing the fully qualified path, but the `Param` type uses methods from the `ServiceParams` trait, which needs to be in scope for those methods to be available.  I don't _think_ there's a way to get access to the methods of a trait that isn't explicitly `use`d